deployment/copilot/devDeploy.sh

```diff
@@ -140,6 +140,9 @@ cdk deploy "*" --tags $TAGS --c domainName="aos-domain" --c engineVersion="OS_2.
 
 # Collect export commands from CDK output, which are needed by Copilot, wrap the commands in double quotes and store them within the "environment" dir
 export_file_path=../../copilot/environments/$COPILOT_DEPLOYMENT_STAGE/envExports.sh
+
+mkdir -p $(dirname "${export_file_path}")
```

Comment:
I assume this will only create the directory if it doesn't exist?

Reply:
That's correct.